### PR TITLE
Switch to using autopilot plugin, with our mods

### PR DESCRIPTION
**Pull request by aeijdenberg**
(Haven't tested yet, so might be best to wait until tomorrow)

It will also mean we'll need to delete the old blue green apps afterwards as a once off.

### CODE REVIEW

**Comment by aeijdenberg** [comment]
We're now using the zero-downtime-push for docs.cloud.gov.au via Circle in this manner and it seems to work well for us there (and having also been using internally for a number of our Cloud support applications).

**Comment by aeijdenberg** [comment]
Note, after this deploys, we should remove the `--show-app-log` as the build logs are public, and there's a slight chance of info leakage.

I am however inclined to keep enabled for the first run, in case we encounter any unexpected issues.

@gordcorp - despite the age of this PR, I did a pass on it earlier today, and I think it's good to go (at least to staging). Would appreciate a review.

File: manifest-production.yml [hunk 1]
```diff
@@ -1,13 +1,9 @@
 ---
-
 path: _site/
+instances: 3
```
Comment (gordcorp) [inline manifest-production.yml]: Wouldn't 2 be enough? I think the main reason for > 1 is so we dont have an outage if we lose a diego-cell?

Comment (aeijdenberg) [reply manifest-production.yml]: Probably 2 is fine.

**Pull request merged**